contracts/utils/structs/Heap.sol

```diff
@@ -283,294 +227,18 @@ library Heap {
      */
     function _siftUp(
         Uint256Heap storage self,
-        uint64 pos,
+        uint64 index,
```

Comment:
We'll need to rename `_siftDown` arguments as well and remove references to `pos`